xds/internal/balancer/clusterresolver/configbuilder_test.go

```diff
@@ -975,3 +1117,77 @@ func testAddrWithAttrs(addrStr string, weight *uint32, priority string, lID *int
 	addr = hierarchy.Set(addr, path)
 	return addr
 }
+
+func TestConvertClusterImplMapToOutlierDetection(t *testing.T) {
+	tests := []struct {
+		name          string
+		ciCfgsMap     map[string]*clusterimpl.LBConfig
+		odCfg         *outlierdetection.LBConfig
+		odCfgsMapWant map[string]*outlierdetection.LBConfig
```

Comment:
Nit: The `want` suffix does not read as well as a `want` prefix. Maybe s/odCfgsMapWant/wantODCfgs/?